sys-auth/sssd/metadata.xml

```diff
@@ -7,6 +7,7 @@
 	</maintainer>
 	<use>
 		<flag name="acl"> Build and use the cifsidmap plugin</flag>
```

Comment:
Please consider adopting the package if you're at all willing. It really needs a dedicated maintainer.

Reply:
Right now I have a little "lab" set up for this - I've got another computer with 3 containers - one running an LDAP server, one running a Kerberos server, and one running a DNS server. I'm willing to adopt and maintain it, although at some point in the future I'd like to see a "production" user maintain this (Ideally 2 - one for AD, and one for non-AD). Right now there are 0 users of this package, the version in Portage is broken: popt had a memory leak, and sssd had a user-after-free. They cancelled each other out and it just happened to work. Once the memory leak was fixed, sssd broke.